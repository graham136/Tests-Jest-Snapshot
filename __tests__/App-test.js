
// React Imports
import React from 'react';

//Custom Imports
import App from '../src/App';

// Test Library Imports
import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer.create(<App />).toJSON();
  expect(tree).toMatchSnapshot();
});

