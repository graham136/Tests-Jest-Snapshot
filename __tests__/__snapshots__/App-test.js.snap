// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  onLayout={[Function]}
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    onMoveShouldSetResponder={[Function]}
    onMoveShouldSetResponderCapture={[Function]}
    onResponderEnd={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderReject={[Function]}
    onResponderRelease={[Function]}
    onResponderStart={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    onStartShouldSetResponderCapture={[Function]}
    style={
      Array [
        Object {
          "flex": 1,
          "flexDirection": "column-reverse",
          "overflow": "hidden",
        },
        Object {
          "backgroundColor": "#000",
        },
      ]
    }
  >
    <View
      style={
        Object {
          "flex": 1,
        }
      }
    >
      <View
        accessibilityElementsHidden={false}
        pointerEvents="auto"
        style={
          Object {
            "backgroundColor": "#E9E9EF",
            "bottom": 0,
            "left": 0,
            "opacity": 1,
            "paddingTop": 0,
            "position": "absolute",
            "right": 0,
            "shadowColor": "black",
            "shadowOffset": Object {
              "height": 0,
              "width": 0,
            },
            "shadowOpacity": 0.2,
            "shadowRadius": 5,
            "top": 0,
            "transform": Array [
              Object {
                "translateX": 0,
              },
              Object {
                "translateY": 0,
              },
            ],
          }
        }
      >
        <View
          onLayout={[Function]}
          style={
            Object {
              "flex": 1,
            }
          }
        >
          <View
            onMoveShouldSetResponder={[Function]}
            onMoveShouldSetResponderCapture={[Function]}
            onResponderEnd={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderReject={[Function]}
            onResponderRelease={[Function]}
            onResponderStart={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            onStartShouldSetResponderCapture={[Function]}
            style={
              Array [
                Object {
                  "flex": 1,
                  "flexDirection": "column-reverse",
                  "overflow": "hidden",
                },
                Object {
                  "backgroundColor": "#000",
                },
              ]
            }
          >
            <View
              style={
                Object {
                  "flex": 1,
                }
              }
            >
              <View
                accessibilityElementsHidden={false}
                pointerEvents="auto"
                style={
                  Object {
                    "backgroundColor": "#E9E9EF",
                    "bottom": 0,
                    "left": 0,
                    "opacity": 1,
                    "paddingTop": 0,
                    "position": "absolute",
                    "right": 0,
                    "shadowColor": "black",
                    "shadowOffset": Object {
                      "height": 0,
                      "width": 0,
                    },
                    "shadowOpacity": 0.2,
                    "shadowRadius": 5,
                    "top": 0,
                    "transform": Array [
                      Object {
                        "translateX": 0,
                      },
                      Object {
                        "translateY": 0,
                      },
                    ],
                  }
                }
              >
                <View
                  accessibilityIgnoresInvertColors={true}
                  style={
                    Object {
                      "alignItems": "center",
                      "flex": 1,
                      "justifyContent": "center",
                    }
                  }
                >
                  <Image
                    source={
                      Object {
                        "testUri": "../../../src/Assets/SARS-BACKGROUND-LOGIN.jpg",
                      }
                    }
                    style={
                      Array [
                        Object {
                          "bottom": 0,
                          "left": 0,
                          "position": "absolute",
                          "right": 0,
                          "top": 0,
                        },
                        Object {
                          "height": undefined,
                          "width": undefined,
                        },
                        undefined,
                      ]
                    }
                  />
                  <View
                    style={
                      Array [
                        Object {
                          "backgroundColor": "#fff",
                          "flex": 1,
                          "height": 1334,
                        },
                        Object {
                          "alignItems": "center",
                          "backgroundColor": "rgba(1, 92, 149, 0.6)",
                          "height": "100%",
                          "justifyContent": "center",
                          "width": "100%",
                        },
                      ]
                    }
                  >
                    <View
                      style={
                        Array [
                          Object {
                            "backgroundColor": "#fff",
                            "borderBottomWidth": 0.5,
                            "borderColor": "#ccc",
                            "borderLeftWidth": 0.5,
                            "borderRadius": 2,
                            "borderRightWidth": 0.5,
                            "borderTopWidth": 0.5,
                            "elevation": 3,
                            "flexWrap": "nowrap",
                            "marginBottom": 5,
                            "marginLeft": 2,
                            "marginRight": 2,
                            "marginTop": 5,
                            "shadowColor": "#000",
                            "shadowOffset": Object {
                              "height": 2,
                              "width": 0,
                            },
                            "shadowOpacity": 0.1,
                            "shadowRadius": 1.5,
                          },
                          Object {
                            "opacity": 1,
                            "width": "40%",
                          },
                        ]
                      }
                    >
                      <View
                        style={
                          Array [
                            Object {
                              "alignItems": "center",
                              "backgroundColor": "#fff",
                              "borderRadius": 2,
                              "flexDirection": "row",
                              "paddingBottom": 10,
                              "paddingLeft": 15,
                              "paddingRight": 15,
                              "paddingTop": 10,
                            },
                            Object {
                              "alignSelf": "center",
                            },
                          ]
                        }
                      >
                        <Image
                          source={
                            Object {
                              "testUri": "../../../src/Assets/SARS-LOGO-BLUE.png",
                            }
                          }
                        />
                      </View>
                      <View
                        style={
                          Array [
                            Object {
                              "alignItems": "center",
                              "backgroundColor": "#fff",
                              "borderRadius": 2,
                              "flexDirection": "row",
                              "paddingBottom": 10,
                              "paddingLeft": 15,
                              "paddingRight": 15,
                              "paddingTop": 10,
                            },
                            Object {
                              "alignSelf": "center",
                              "color": "rgb(1, 92, 149)",
                            },
                          ]
                        }
                      >
                        <Text
                          style={
                            Object {
                              "color": "#000",
                              "fontFamily": "System",
                              "fontSize": 16,
                            }
                          }
                          uppercase={false}
                        >
                          Customs Manual Case Creation
                        </Text>
                      </View>
                      <View
                        style={
                          Object {
                            "alignItems": "center",
                            "backgroundColor": "#fff",
                            "borderRadius": 2,
                            "flexDirection": "row",
                            "paddingBottom": 10,
                            "paddingLeft": 15,
                            "paddingRight": 15,
                            "paddingTop": 10,
                          }
                        }
                      >
                        <RCTScrollView
                          automaticallyAdjustContentInsets={false}
                          contentContainerStyle={
                            Array [
                              Object {
                                "padding": undefined,
                              },
                              undefined,
                            ]
                          }
                          contentInset={
                            Object {
                              "bottom": 0,
                            }
                          }
                          enableAutomaticScroll={true}
                          enableOnAndroid={false}
                          enableResetScrollToCoords={true}
                          extraHeight={75}
                          extraScrollHeight={0}
                          getScrollResponder={[Function]}
                          handleOnScroll={[Function]}
                          keyboardDismissMode="interactive"
                          keyboardOpeningTime={250}
                          keyboardShouldPersistTaps="handled"
                          keyboardSpace={0}
                          onScroll={[Function]}
                          resetKeyboardSpace={[Function]}
                          resetScrollToCoords={
                            Object {
                              "x": 0,
                              "y": 0,
                            }
                          }
                          scrollEventThrottle={1}
                          scrollForExtraHeightOnAndroid={[Function]}
                          scrollIntoView={[Function]}
                          scrollToEnd={[Function]}
                          scrollToFocusedInput={[Function]}
                          scrollToPosition={[Function]}
                          showsVerticalScrollIndicator={true}
                          style={
                            Object {
                              "backgroundColor": "transparent",
                              "flex": 1,
                            }
                          }
                          update={[Function]}
                          viewIsInsideTabBar={false}
                        >
                          <View>
                            <View
                              style={Object {}}
                            >
                              <View
                                accessible={true}
                                isTVSelectable={true}
                                onResponderGrant={[Function]}
                                onResponderMove={[Function]}
                                onResponderRelease={[Function]}
                                onResponderTerminate={[Function]}
                                onResponderTerminationRequest={[Function]}
                                onStartShouldSetResponder={[Function]}
                                style={
                                  Object {
                                    "alignItems": "center",
                                    "backgroundColor": "transparent",
                                    "borderBottomWidth": 1,
                                    "borderColor": "#D9D5DC",
                                    "borderLeftWidth": 0,
                                    "borderRadius": undefined,
                                    "borderRightWidth": 0,
                                    "borderTopWidth": 0,
                                    "borderWidth": undefined,
                                    "flexDirection": "row",
                                    "marginLeft": 15,
                                    "opacity": 1,
                                  }
                                }
                              >
                                <TextInput
                                  allowFontScaling={true}
                                  editable={true}
                                  placeholder="Username"
                                  placeholderTextColor="#575757"
                                  rejectResponderTermination={true}
                                  style={
                                    Object {
                                      "color": "#000",
                                      "flex": 1,
                                      "fontSize": 17,
                                      "height": 50,
                                      "paddingLeft": 5,
                                      "paddingRight": 5,
                                      "top": 1.5,
                                    }
                                  }
                                  underlineColorAndroid="rgba(0,0,0,0)"
                                />
                                <Text
                                  allowFontScaling={false}
                                  style={
                                    Array [
                                      Object {
                                        "color": undefined,
                                        "fontSize": 12,
                                      },
                                      undefined,
                                      Object {
                                        "fontFamily": "FontAwesome",
                                        "fontStyle": "normal",
                                        "fontWeight": "normal",
                                      },
                                      Object {},
                                    ]
                                  }
                                >
                                  
                                </Text>
                              </View>
                              <View
                                accessible={true}
                                isTVSelectable={true}
                                onResponderGrant={[Function]}
                                onResponderMove={[Function]}
                                onResponderRelease={[Function]}
                                onResponderTerminate={[Function]}
                                onResponderTerminationRequest={[Function]}
                                onStartShouldSetResponder={[Function]}
                                style={
                                  Object {
                                    "alignItems": "center",
                                    "backgroundColor": "transparent",
                                    "borderBottomWidth": 1,
                                    "borderColor": "#D9D5DC",
                                    "borderLeftWidth": 0,
                                    "borderRadius": undefined,
                                    "borderRightWidth": 0,
                                    "borderTopWidth": 0,
                                    "borderWidth": undefined,
                                    "flexDirection": "row",
                                    "marginLeft": 0,
                                    "opacity": 1,
                                    "paddingLeft": 15,
                                  }
                                }
                              >
                                <TextInput
                                  allowFontScaling={true}
                                  editable={true}
                                  placeholder="Password"
                                  placeholderTextColor="#575757"
                                  rejectResponderTermination={true}
                                  style={
                                    Object {
                                      "color": "#000",
                                      "flex": 1,
                                      "fontSize": 17,
                                      "height": 50,
                                      "paddingLeft": 5,
                                      "paddingRight": 5,
                                      "top": 1.5,
                                    }
                                  }
                                  underlineColorAndroid="rgba(0,0,0,0)"
                                />
                                <Text
                                  allowFontScaling={false}
                                  style={
                                    Array [
                                      Object {
                                        "color": undefined,
                                        "fontSize": 12,
                                      },
                                      undefined,
                                      Object {
                                        "fontFamily": "FontAwesome",
                                        "fontStyle": "normal",
                                        "fontWeight": "normal",
                                      },
                                      Object {},
                                    ]
                                  }
                                >
                                  
                                </Text>
                              </View>
                            </View>
                          </View>
                        </RCTScrollView>
                      </View>
                      <View
                        style={
                          Object {
                            "alignItems": "center",
                            "backgroundColor": "#fff",
                            "borderRadius": 2,
                            "flexDirection": "row",
                            "paddingBottom": 10,
                            "paddingLeft": 15,
                            "paddingRight": 15,
                            "paddingTop": 10,
                          }
                        }
                      >
                        <View
                          style={
                            Object {
                              "alignItems": "flex-start",
                              "alignSelf": "stretch",
                              "flex": 1,
                            }
                          }
                        >
                          <View
                            accessible={true}
                            isTVSelectable={true}
                            onResponderGrant={[Function]}
                            onResponderMove={[Function]}
                            onResponderRelease={[Function]}
                            onResponderTerminate={[Function]}
                            onResponderTerminationRequest={[Function]}
                            onStartShouldSetResponder={[Function]}
                            style={
                              Object {
                                "alignItems": "center",
                                "alignSelf": "center",
                                "backgroundColor": "rgb(1, 92, 149)",
                                "borderBottomWidth": null,
                                "borderColor": "#007aff",
                                "borderLeftWidth": null,
                                "borderRadius": 5,
                                "borderRightWidth": null,
                                "borderTopWidth": null,
                                "borderWidth": undefined,
                                "elevation": 2,
                                "flexDirection": "row",
                                "height": 45,
                                "justifyContent": "space-between",
                                "opacity": 1,
                                "paddingBottom": 6,
                                "paddingTop": 6,
                                "shadowColor": undefined,
                                "shadowOffset": undefined,
                                "shadowOpacity": undefined,
                                "shadowRadius": undefined,
                              }
                            }
                          >
                            <Text
                              style={
                                Object {
                                  "backgroundColor": "transparent",
                                  "color": "#fff",
                                  "fontFamily": "System",
                                  "fontSize": 16.5,
                                  "marginLeft": 0,
                                  "marginRight": 0,
                                  "paddingLeft": 16,
                                  "paddingRight": 16,
                                }
                              }
                              uppercase={false}
                            >
                              Login
                            </Text>
                          </View>
                        </View>
                      </View>
                    </View>
                  </View>
                </View>
              </View>
            </View>
            <View
              onLayout={[Function]}
              pointerEvents="box-none"
              style={
                Object {
                  "left": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                }
              }
            >
              <View
                style={
                  Object {
                    "backgroundColor": "#FFF",
                    "transform": Array [
                      Object {
                        "translateX": -750,
                      },
                    ],
                  }
                }
              >
                <View
                  onLayout={[Function]}
                  pointerEvents="box-none"
                  style={
                    Object {
                      "backgroundColor": "#FFF",
                      "borderBottomColor": "#A7A7AA",
                      "borderBottomWidth": 0.5,
                      "height": 64,
                      "paddingBottom": 0,
                      "paddingLeft": 0,
                      "paddingRight": 0,
                      "paddingTop": 20,
                    }
                  }
                >
                  <View
                    style={
                      Object {
                        "flex": 1,
                      }
                    }
                  />
                </View>
              </View>
            </View>
          </View>
        </View>
      </View>
    </View>
    <View
      onLayout={[Function]}
      pointerEvents="box-none"
      style={
        Object {
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        }
      }
    >
      <View
        style={
          Object {
            "backgroundColor": "#FFF",
            "transform": Array [
              Object {
                "translateX": -750,
              },
            ],
          }
        }
      >
        <View
          onLayout={[Function]}
          pointerEvents="box-none"
          style={
            Object {
              "backgroundColor": "#FFF",
              "borderBottomColor": "#A7A7AA",
              "borderBottomWidth": 0.5,
              "height": 64,
              "paddingBottom": 0,
              "paddingLeft": 0,
              "paddingRight": 0,
              "paddingTop": 20,
            }
          }
        >
          <View
            style={
              Object {
                "flex": 1,
              }
            }
          />
        </View>
      </View>
    </View>
  </View>
</View>
`;
